com.q3rally.Q3Rally.yaml

```diff
@@ -28,14 +28,15 @@ modules:
       - mv ${FLATPAK_DEST}/bin/*.so ${FLATPAK_DEST}/lib/q3rally/
       - install -Dm0644 engine/misc/quake3.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/q3rally.svg
       - install -Dm0644 engine/misc/setup/q3rally.desktop ${FLATPAK_DEST}/share/applications/q3rally.desktop
-      - install -Dm0644 engine/misc/setup/${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - # engine/misc/setup/${FLATPAK_ID}.metainfo.xml
+      - install -Dm0644 com.q3rally.Q3Rally.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
     sources:
       - type: git
         url: https://github.com/Q3Rally-Team/q3rally.git
         tag: v0.4
         commit: e8de4d00a02042d577e96efbde242f796546a1c9
-      - type: patch
-        path: appstream.patch
+      - type: file
+        path: com.q3rally.Q3Rally.metainfo.xml
```

Comment:
Can you please include the second patch here?

Reply:
Okay, I added it as a second patch.